assets/app/features/upgrade/controllers/upgrade-backup.controller.spec.js

```diff
@@ -161,21 +148,15 @@ describe('Upgrade Flow - Backup Controller', function() {
                     });
 
                     it('uses the default name for the file', function () {
-                        expect(downloadBackupMock.download).toEqual('crowbarBackup');
+                        expect(FileSaver.saveAs).toHaveBeenCalledWith(
+                          mockedDownloadResponse.data, 'crowbarBackup'
```

Comment:
Since this test is about the filename, I'd put jasmine.anything() as the first argument. The data can be a separate test.